cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -6016,14 +6016,23 @@
   </property>
 
   <property>
-    <name>internal.router.enabled</name>
+    <name>internal.router.clients.enabled</name>
```

Comment:
Keep it as singular `client` instead of plural.

Reply:
Changed.